### DISCUSSION TITLE
Facebook disconnect after connecting.

### DISCUSSION THREAD

**Discussion by juniorgalhard** [chatwoot/chatwoot#5011] [General]
hello person.
I did the facebook configuration and I can log in and select the page I want, but right after finishing the configuration, it says that the page has been discounted.

in facebook app config in OAuth url, configure accurate with Chatwoot url?

<img width="935" alt="Screen Shot 2022-07-10 at 22 12 54" src="https://user-images.githubusercontent.com/11300948/178170981-d790a17d-4c5a-4167-90da-6ae7f77b83ee.png">

**Comment by sojan-official** [comment]
@juniorgalhard  can you share the server logs?

**Comment by juniorgalhard** [comment]
Hello @sojan-official 

some images about the error.

![04](https://user-images.githubusercontent.com/11300948/188203003-44a97dd8-6924-4868-a67c-71c4d2786098.png)
![02](https://user-images.githubusercontent.com/11300948/188203007-2a68d918-1076-4cae-a43d-f6fac50c87bd.png)
![01](https://user-images.githubusercontent.com/11300948/188203010-5dba1b86-74ab-4ec2-a330-63f656112f2a.png)

**Reply by sojan-official** [reply]
Could you check the server logs for any errors ?
